src/Microsoft.NetCore.Analyzers/Core/Security/SystemSecurityCryptographyResources.resx

```diff
@@ -132,6 +132,12 @@
   <data name="DefinitelyInstallRootCertMessage" xml:space="preserve">
     <value>Adding certificates to the operating system's trusted root certificates increases the risk of incorrectly authenticating an illegitimate certificate</value>
   </data>
+  <data name="DefinitelyUseWeakKDFInsufficientIterationCount" xml:space="preserve">
+    <value>Do Not Use Weak Key Derivation Function With Insufficient Iteration Count</value>
+  </data>
+  <data name="DefinitelyUseWeakKDFInsufficientIterationCountMessage" xml:space="preserve">
+    <value>Default iteration count 1000 needs to be overwritten in this object creation. Use at least 100k iterations when deriving cryptographic key from password</value>
```

Comment:
>Default iteration count 1000 needs to be overwritten in this object creation. Use at least 100k iterations when deriving cryptographic key from password [](start = 11, length = 152)

Use at least {0} iterations when deriving a cryptographic key from a password. By default, Rfc2898DeriveByte's IterationCount is only {1}.

(Put the numbers in string format arguments so they get localized correctly.)